### ISSUE TITLE
Simple project health measure by counting PRs and issues

### ISSUE THREAD

**Issue by mehdisadeghi** [sindresorhus/refined-github#2064]
TL;DR: add a measure for project health by checking issues, PRs and comments.

Example URL:

Probaby in front of the repo name.

Everytime I want to choose a new dependency for a project, I take a few steps (and check red flags) to make sure the project is being maintained, has a healthy community and volonteer contributions are being respected.

1. I look at the number of open PRs. If there are too many open PRs, the project is probably unmaintained.
2. Then I look at the PRs in more detail. If there are no comments from maintainers, then the project is unmaintained. At this point I close the browser tab and move on. I don't consider failing PRs.
3. There are comments from maintainers but they are picky and keep rejecting PRs. i.e. PRs pass tests but are not accepted. This is another red flag that tells me someone is attached to their project and it is more of a personal hobby than a community project. I often move on.
4. I look at the issues, if there are hundreds of open issues, depending on the size of the project it might be a red flag. Perhaps the project has many bugs, or is doing too many different things at the same time or designed poorly. This is not as bad as dangling PRs but still makes me alert.

I would like to know your thoughts on such a feature. Do you think we can come up with a formula of sorts to calculate such a measure?

Thanks for reading.

**Comment by notlmn** [comment]
Possible duplicate of #1920.

As already mentioned in https://github.com/sindresorhus/refined-github/issues/1920#issuecomment-482928852, the fact that a repository has too less or too many open pull-requests or issues doesn't make a project less useful than something else.

There can be various reasons for a repository to be inactive, ranging from the project being matured enough (feature complete in some cases) to cases where the maintainer do not have enough time to get back to the contributions.

People have various reasons to not get back to their users, and that does not make a maintainer of the project bad or ignorant.

I've said this before, IMHO, marking a repository as inactive would only make the reader less interested in the repository. And I don't see how helpful that feature is to the open-source community or even consider something like that a "feature" in the first place.

Thanks!

**Comment by bfred-it** [comment]
Indeed a duplicate of #1920

**Comment by mehdisadeghi** [comment]
> People have various reasons to not get back to their users, and that does not make a maintainer of the project bad or ignorant.

This is not about judging people behind a repository. They have already done a great job by releasing their code to the community. And I appreciate that.

> marking a repository as inactive would only make the reader less interested in the repository.

I'm indeed less interested in repositories that are unmaintained or their owners hardly accept PRs. The whole point is about realizing this quickly. I happily use and support some projects that have "quickly" turned down my PRs and closed my issues, compared to some that even after months or years haven't even replied.

And let's not call it inactive, but "Engagement Metric".

Thanks for your time!
